src/main/java/ay2021s1_cs2103_w16_3/finesse/logic/commands/AddExpenseCommand.java

```diff
@@ -21,8 +21,9 @@
             + "Parameters: "
             + PREFIX_TITLE + "TITLE "
             + PREFIX_AMOUNT + "AMOUNT "
-            + PREFIX_DATE + "DATE "
+            + "[" + PREFIX_DATE + "DATE] "
             + "[" + PREFIX_CATEGORY + "CATEGORY]...\n"
+            + "Inputting the date is optional. If no input is given for d/DATE, current date will be used.\n"
```

Comment:
```suggestion
            + "Inputting the date is optional. If no input is given for d/DATE, the current date will be used.\n"
```

Same for the other instances of this sentence.